app/main.py

```diff
@@ -65,8 +66,20 @@ async def initialize_and_tick_agent(task_prompt: str, agent: GPT4Agent = Depends
         agent.initialize(task_prompt)
         agent.tick()
 
+
+    @app.post("/phase_out")
```

Comment:
phase in / phase out endpoints. Phase in a WIP / needs more tests and slightly different initialization / construction pattern.